gitpod-io/website
# add jb product docs generator

## Description


It allows to edit a single md file to adjust docs of all JB products.

## Related Issue(s)

Fixes #

## How to test


- Start a gitpod workspace and run `yarn generate:jb` you should not see any changes.
- Edit jp-product.md and try again, you should see that all products are updated.

## Release Notes

```release-note
NONE
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1946"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
